cluster/examples/cockroach/class.yaml

```diff
@@ -1,24 +1,9 @@
-apiVersion: v1
-kind: Namespace
-metadata:
-  name: app-project1-dev
----
-apiVersion: database.crossplane.io/v1alpha1
-kind: PostgreSQLInstanceClass
-metadata:
-  name: postgresql-standard
-  namespace: app-project1-dev
-classRef:
-  kind: CockroachClusterClass
-  apiVersion: database.rook.crossplane.io/v1alpha1
-  name: cockroach-standard
-  namespace: rook-infra-dev
----
 apiVersion: database.rook.crossplane.io/v1alpha1
 kind: CockroachClusterClass
 metadata:
   name: cockroach-standard
-  namespace: rook-infra-dev
+  labels:
+    default: "true"
 specTemplate:
   providerRef:
     name: demo-k8s-provider
```

Comment:
You'll  need to remove the namespace from this and all other `providerRef` fields.

Reply:
I suspect you'll also need a `.specTemplate.writeConnectionSecretsToNamespace`, which is required (even if the claim doesn't write connection secrets).